sp041_20191205T1757.cdl

```diff
@@ -332,4 +332,46 @@ data:
  PLATFORM_MODEL = "Spray" ;
 
  WMO_IDENTIFIER = "4801948" ;
+
+group: CTD {
+
+  // group attributes:
+  		:long_name = "CTD sensor" ;
+  		:type = "CTD" ;
+  		:type_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:maker = "Seabird" ;
+  		:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:model = "SBE 41CP" ;
```

Comment:
Thinking needs an additional attribute to include the id of the particular concept used. 
E.g. URL/URI for this example is https://vocab.nerc.ac.uk/collection/L22/current/TOOL0669/
or can use this as the identifier SDN:L22::TOOL0669
The link is probably the preference if size isn't an issue

Reply:
Is it OK to use the title of the item as done with type and maker? I'm updating the cdl here to confirm with you.

Reply:
I understand the point of Emma. 
If we use the title, then it is not machine readable, if we only use the URI it is not human readable...
The file should be both

I think "maker"/"type"/"model" should be human readable so we should use the title of the concept.
while "maker_vocabulary"/"type_vocabulary"/"model_vocabulary" should be machine readable an then use the url to the concept "https://vocab.nerc.ac.uk/collection/L22/current/TOOL0669/".
So I would suggest : 
 		:model = "SBE 41CP" ;
  		:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL0669/" ;

or additing the UUID to link to the concept: 
 		:model = "SBE 41CP" ;
  		:model_vocabulary = "http://vocab.nerc.ac.uk/collection/L22/current/" ;
 		:model_concept_UUID = "SDN:L22::TOOL0669" ;

Reply:
Agree with the idea of both human and machine readable. Just an edit on the above would be: 

:model = "SBE 41CP" ;
:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL0669" ;
:UUID = "TOOL0669_75" ;

I don't think it is useful to either machine or human to have the static collection URL (https://vocab.nerc.ac.uk/collection/L22/current/) , you might as well point to the actual concept this model is referring to. This gives the machine one link access. 

The above block gives a user the title as human readable, the machine access to the model concept and then both have access via the UUID to the specific instance (serial) of this model. 

The value of the UUID could be a URI/URL if we want it to be.

Reply:
Sounds good. @emmerbodc , How was the "75" part of the `UUID` defined? How will we guarantee that this `UUID` will be unique among different providers? Can we trust the maker's serial number?

Reply:
This is related to the conversation below, where it is considered to drop UUID completely. This doesn't mean that in the data pipeline wouldn't use it, but we wouldn't require the data providers to add it.

Reply:
I think that the resolution was to have it as a suggested field and recommend using concept + serial number. If you disagree that this is a resolved matter, please re-open this conversation of just let me know. Thanks @emmerbodc and @vturpin